Poolmgr concurrency (#215)
fission/docs.fission.io: docs/content/en/docs/usage/executor.en.md

### Poolmgr (Pool-based executor)

You can create a function like following to use poolmgr as backend executor. You can use the `concurrency` field to control the 

> maximum number of concurrent pod specialization(default 5) to serve requests. As of now, `concurrency` field is only supported by poolmgr.

Reviewer: ```suggestion
maximum number of concurrent pod specialization(default 5) to serve requests. The `concurrency` field is only supported by poolmgr.
```
